docs/diagrams/AddEventActivityDiagram.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+start
+:User executes AddEventCommand;
+
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if () then
+    :Throw InvalidCommandException;
+else ([ValidSyntax])
+if () then
```

Comment:
Appreciate the decision to leave out the `[Else]` condition! Improves readability of the diagram.

Good job!